backend/cinema_service/settings.py

```diff
@@ -89,9 +91,10 @@
     "default": {
         "ENGINE": "django.db.backends.postgresql",
         "HOST": os.environ["POSTGRES_HOST"],
-        "NAME": os.environ["POSTGRES_DB"],
+        "NAME": os.environ["POSTGRES_NAME"],
```

Comment:
Why did you change this?

Reply:
ohh, sorry, i made changes in .env and didn't reflect those changes in .enc_sample.